pkgs/tools/wayland/swayr/icon-paths.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/src/config.rs b/src/config.rs
+index de7d04b..d58111c 100644
+--- a/src/config.rs
++++ b/src/config.rs
+@@ -197,6 +197,12 @@ impl Default for Format {
+             ),
+             urgency_end: Some("</span>".to_string()),
+             icon_dirs: Some(vec![
++                "/run/current-system/sw/share/icons/hicolor/scalable/apps".to_string(),
++                "/run/current-system/sw/share/icons/hicolor/48x48/apps".to_string(),
++                "/run/current-system/sw/share/pixmaps".to_string(),
++                "/home/YOURUSER/.nix-profile/share/icons/hicolor/scalable/apps".to_string(),
```

Comment:
how does `/home/YOURUSER/` work? Does swayr substitute `YOURUSER`?

Reply:
users were supposed to replace it by themselves. i asked for the ability to use env vars but seems like we can use ~ https://todo.sr.ht/~tsdh/swayr/3#event-90358

Reply:
tsdh added support for tilde https://git.sr.ht/~tsdh/swayr/commit/9e6ed161925e1f45a3f58d6770a47b2f141efe60

Reply:
tsdh will make a release soon

Reply:
:+1: ok I think we can merge this after the next release since it'll be soon, just so everything is working